### PR TITLE
[inbound calling] incoming call conceptual docs

### PR DESCRIPTION
**Pull request by dmceachernmsft**
# What

Add conceptual docs for incoming calls
# Why

provides complete docs for feature

# How Tested

Validate in storybook

### CODE REVIEW

File: packages/storybook/stories/INTERNAL/IncomingCallNotification/InboundCalling.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  parameters={{ previewTabs: { canvas: { disable: true, hidden: true } } }}
+/>
+
+# Inbound Calling
+
+Azure Communication Services UI Library is adding support for Inbound Calling. With this new feature, users will
+be able to see and monitor new incoming calls when signed in to ACS or Teams as a Communication as a Teams User.
+Users will be able to Accept with Audio, Accept with video, and reject incoming calls.
+
+Enabling this feature will allow them see multiple incoming calls allowing the user to service multiple people
```
Comment (mgamis-msft) [inline packages/storybook/stories/INTERNAL/IncomingCallNotification/InboundCalling.stories.mdx:15]: typo: "... allow them to see ..."

File: packages/storybook/stories/INTERNAL/IncomingCallNotification/InboundCalling.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  id="inboundcalling"
+  title="Concepts/InboundCalling"
+  parameters={{ previewTabs: { canvas: { disable: true, hidden: true } } }}
+/>
+
+# Inbound Calling
+
+Azure Communication Services UI Library is adding support for Inbound Calling. With this new feature, users will
+be able to see and monitor new incoming calls when signed in to ACS or Teams as a Communication as a Teams User.
+Users will be able to Accept with Audio, Accept with video, and reject incoming calls.
```
Comment (mgamis-msft) [inline packages/storybook/stories/INTERNAL/IncomingCallNotification/InboundCalling.stories.mdx:13]: nit: why is Accept capitalized but reject is not?

File: packages/storybook/stories/INTERNAL/IncomingCallNotification/InboundCalling.stories.mdx [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+Enabling this feature will allow them see multiple incoming calls allowing the user to service multiple people
+at once best supporting the people that want to get in touch. In any scenario when your user’s are logged in you
```
Comment (mgamis-msft) [inline packages/storybook/stories/INTERNAL/IncomingCallNotification/InboundCalling.stories.mdx:16]: maybe "...assist multiple people simultaneously and better support those trying to reach them."

File: packages/storybook/stories/INTERNAL/IncomingCallNotification/IncomingCallNotification.stories.tsx [hunk 4]
```diff
@@ -2,39 +2,41 @@
...
       <Canvas mdxSource={IncomingCallNotificationText}>
         <IncomingCallNotificationExample />
       </Canvas>
       <Heading>Styling</Heading>
       <Description>
-        Depending on your use of incoming call notification you might want to customize it to your needs. The component
-        includes API's to adjust the CSS on many different elements of the component. Below is an example of how you can
-        customize incoming call notification.
+        Depending on your use of `IncomingCallNotification` and the `IncomingCallStack` you might want to customize your
+        notifications appearence to match your needs. Both components includes API's to adjust the CSS on many different
```
Comment (mgamis-msft) [inline packages/storybook/stories/INTERNAL/IncomingCallNotification/IncomingCallNotification.stories.tsx:34]: type: appearance

File: packages/storybook/stories/INTERNAL/IncomingCallNotification/IncomingCallNotification.stories.tsx [hunk 5]
```diff
@@ -2,39 +2,41 @@
...
     <>
-      <Title>Incoming Call Notification Component</Title>
+      <Title>IncomingCallNotification</Title>
       <SingleLineBetaBanner />
       <Description>
-        The incoming Call notification component is used to provide information to your users when they are recieving a
-        call. This UI component is to be used to represent a single incoming call.
+        This component is a representation of an incoming call. It will show the name of the caller and allow you to
+        accept the call with either audio or video or reject the call. This component is used by the `IncomingCallStack`
+        for each of the calls held in it's state.
```
Comment (mgamis-msft) [inline packages/storybook/stories/INTERNAL/IncomingCallNotification/IncomingCallNotification.stories.tsx:26]: should be "... held in its state."

File: packages/storybook/stories/INTERNAL/IncomingCallNotification/InboundCalling.stories.mdx [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+are able to Accept or reject calls based on your user’s needs.
+
+## Incorporating Inbound Calling into your Experience
+
+Currently the UI library exports a series of components like the [Video
+Gallery](./?path=/docs/ui-components-videogallery--video-gallery) or
+[CallControls](./?path=/docs/ui-components-controlbar--control-bar) that allow you to build a Video calling
+experience that you can join through our [stateful client](./?path=/docs/quickstarts-statefulcallclient--page).
+With the introduction of these components we are also allowing you to attach to your experience a way to have
```
Comment (mgamis-msft) [inline packages/storybook/stories/INTERNAL/IncomingCallNotification/InboundCalling.stories.mdx:25]: nit: I would add a comma after "With the introduction of these components"

**Review by mgamis-msft — APPROVED** [review]

**Review by PorterNan — APPROVED** [review]

File: packages/storybook/stories/INTERNAL/IncomingCallNotification/InboundCalling.stories.mdx [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+<Meta
+  id="inboundcalling"
+  title="Concepts/InboundCalling"
+  parameters={{ previewTabs: { canvas: { disable: true, hidden: true } } }}
+/>
+
+# Inbound Calling
+
+Azure Communication Services UI Library is adding support for Inbound Calling. With this new feature, users will
+be able to see and monitor new incoming calls when signed in to ACS or Teams as a Communication as a Teams User.
```
Comment (PorterNan) [inline packages/storybook/stories/INTERNAL/IncomingCallNotification/InboundCalling.stories.mdx:12]: As an Azure Communication User or a Teams User

File: packages/storybook/stories/INTERNAL/IncomingCallNotification/InboundCalling.stories.mdx [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+Users will be able to accept with audio, accept with video, and reject incoming calls.
+
+Enabling this feature will allow them to see multiple incoming calls allowing the user to service multiple people
```
Comment (PorterNan) [inline packages/storybook/stories/INTERNAL/IncomingCallNotification/InboundCalling.stories.mdx:15]:
```suggestion
Enabling this feature will allow them to see multiple incoming calls, allowing the user to service multiple people
```

File: packages/storybook/stories/INTERNAL/IncomingCallNotification/InboundCalling.stories.mdx [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+at once best supporting the people that want to get in touch. In any scenario when your user’s assist multiple people simultaneously and better support those trying to reach them. In any scenario, when your users are logged in, you are able to accept or reject based on your users' needs.
```
Comment (PorterNan) [inline packages/storybook/stories/INTERNAL/IncomingCallNotification/InboundCalling.stories.mdx:16]: "Enabling this feature will allow them to see multiple incoming calls allowing the user to service multiple people at once best supporting the people that want to get in touch. "

Just feel a bit hard to split this sentence when reading, maybe adding some comma or "and"

Comment (PorterNan) [reply packages/storybook/stories/INTERNAL/IncomingCallNotification/InboundCalling.stories.mdx:16]: "In any scenario when your user’s assist multiple people simultaneously and better support those trying to reach them. In any scenario, when your users are logged in, you are able to accept or reject based on your users' needs."

This one looks a bit repetitive to previous sentences

**Pull request merged**